pkgs/data/themes/cdetheme/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, python2Packages }:
+
+stdenv.mkDerivation rec {
+  pname = "cdetheme";
```

Comment:
I would name at least the derivation following the upstream project name: `cde-motif-theme`

Reply:
@romildo Done, thanks!